feathr_project/feathr/api/app/core/error_code.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+class ErrorCode(Enum):
+    invalid_code = 1
```

Comment:
I thought REST API have a lot error codes?

Reply:
Can you explain a bit on how it works? I thought HTTP also has something representing invalid code etc.